frontend/src/components/workspace/CreateApplicationButton.tsx

```diff
@@ -23,9 +28,9 @@ const CreateApplicationButton = () => {
       .finally(() => {
         setLoading(false);
       });
-  }, [router]);
+  }, [router, user]);
   return (
-    <Button onClick={handleSubmit} type="submit">
+    <Button disabled={disabled} onClick={handleSubmit} type="submit">
```

Comment:
Disable button if not logged in. This is temporary and will be updated in the next ticket to redirect unauth correctly.